app/routes/docs.tsx

```diff
@@ -28,7 +28,7 @@ const DirectoryLink = ({
     </div>
   ) : (
     <Link
-      to={d.path.replace(/\/?index$/, "")}
+      to={`/${d.path.replace(/\/?index$/, "")}`}
```

Comment:
this fixed my `docs/docs` issue but maybe check on your end as there seems to be conflicting results?

Reply:
I think this would break relative paths, e.g. `../getting_started`

Though this points me to `d.path`. I bet there are places in our `loader` where we are replacing `/` when we should be replacing `[/\\]`

Reply:
It was a windows thing.  My f.path has `\` - `docs\\basics\\references.md`